.azure-pipelines/env.yml

```diff
@@ -212,8 +214,12 @@ jobs:
               PUBLISH_DOCS=false
           fi
       fi
+      if [[ -z "$POSTSUBMIT" ]]; then
+          PUBLISH_GITHUB_RELEASE_DRY_RUN=true
+      fi
```

Comment:
what will happen if the `-dev` is removed and it's postsubmit but the tag is not ready? will it run the github release action?

Reply:
> what will happen if the -dev is removed and it's postsubmit 

it publishes

>  but the tag is not ready?

then a mistake has happened

the automation is based upon specifically those criteria - ie non-dev + postsubmit = release